javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,67 @@
+const btnStart = document.getElementById("btnStartGame");
+const paraS = document.getElementById("paraS");
+const paraL = document.getElementById("paraL");
+const confettiS = document.getElementById("canvasS");
+const confettiL = document.getElementById("canvasL");
+
+let confetti;
+
+let keyS = 0;
+let keyL = 0;
+
+function handleKeyPress(event) {
+  if (event.key === "s") {
+    keyS++;
+    paraS.innerHTML = keyS;
+  } else if (event.key === "l") {
+    keyL++;
+    paraL.innerHTML = keyL;
+  }
+}
+
+function gameStart() {
+  try {
+    if (confetti) {
+      confetti.clear();
+    }
+  } catch (error) {}
+  let interval = parseInt(document.getElementById("timeInterval").value);
+  paraS.innerHTML = "";
+  paraL.innerHTML = "";
+  if (interval) {
+    window.addEventListener("keypress", handleKeyPress);
+
+    const timeInterval = setInterval(function () {
+      if (interval <= 0) {
+        clearInterval(timeInterval);
+        document.getElementById("countDown").innerHTML = "Game Over";
+      } else {
+        document.getElementById("countDown").innerHTML =
+          interval + " seconds remaining";
+      }
+      interval -= 1;
+    }, 1000);
+
+    setTimeout(function () {
+      window.removeEventListener("keypress", handleKeyPress);
+      if (keyS === keyL && keyS != 0 && keyL != 0) {
```

Comment:
This is the same as `keyS === keyL && keyS && keyL` to make it shorter. Also, you should have used `!==` instead of `!=`.